src/nodes.jl

```diff
@@ -616,7 +616,8 @@ Print a simplified string representation of the `CollectNode` with its nodes Vec
 result DeferredFuture RemoteChannel parameters, and its label.
 """
 function Base.show(io::IO, node::CollectNode)
-    print(io, "$(typeof(node).name.name)($(node.nodes),$(node.result),\"$(node.label)\")")
+    nodes = "DispatchNode[$(join(node.nodes, ","))]"
```

Comment:
To use `join` in a `show` method, use this method: 

```markdown
    join(io, items, delim, [last])

Print elements of items to io with delim between them. If last is specified, it is used as the final delimiter
instead of delim.
```